Software/src/Settings.cpp

```diff
@@ -155,7 +155,7 @@ static const QString OverBrighten = "Grab/OverBrighten";
 static const QString IsMinimumLuminosityEnabled = "Grab/IsMinimumLuminosityEnabled";
 static const QString IsDx1011GrabberEnabled = "Grab/IsDX1011GrabberEnabled";
 static const QString IsDx9GrabbingEnabled = "Grab/IsDX9GrabbingEnabled";
-static const QString IsApplyGammaRampEnabled = "Grab/IsApplyGammaRampEnabled";
+static const QString IsApplyBlueLightReductionEnabled = "Grab/IsApplyBlueLightReductionEnabled";
```

Comment:
Yeah, true. I'm tempted to put back the old name to avoid adding migration code... ooor what if we put in the release notes something like "btw this option will be reset to it's default (ON), uncheck if needed" ?

Reply:
Damn I was too fast merging. The problem is that, especially with automatic update, most users do not read change notes. I'm in favor of migration code, but I'm not terribly against having the string mismatch the variable name.

Reply:
There is in fact a settings migration. Unfortunately it's only for the main config file

Reply:
Yeah I saw it shortly after and was excited for half a second before realizing :/